R/install_spark_versions.R

```diff
@@ -1,149 +0,0 @@
-.globals <- new.env(parent = emptyenv())
```

Comment:
Isn't this `.globals` object also used for extension packages (e.g. https://github.com/rstudio/sparklyr/blob/06d0a96fedecefd76d7a94c2a3ee47d6bf70baf4/R/spark_extensions.R#L15-L24)?

I think this needs to be re-defined in a separate file (perhaps in a new file called `aaa.R`, named as such just so it's sourced first by R)